public_profiler/v3.5-DD/ogl/skills.js

```diff
@@ -472,21 +477,35 @@ var skillsHelpURL = Object.assign(
   // Next the extra help:
   extraSpellHelp ? {
     // Feats:
+    'brutal throw':                         baseExtraHelpURL + 'Brutal_Throw',
+    'coordinated strike':                   baseExtraHelpURL + 'Coordinated_Strike',
+    'deadly defense':                       baseExtraHelpURL + 'Deadly_Defense',
+    'divine spell power':                   baseExtraHelpURL + 'Divine_Spell_Power',
     'expeditious dodge':                    baseExtraHelpURL + 'Expeditious_Dodge',
+    'scumpt spell':                         baseExtraHelpURL + 'Sculpt_Spell',
+    'versatile spellcaster':                baseExtraHelpURL + 'Versatile_Spellcaster',
 
-    // Dervish special abilities.
+    // Dervish class features.
```

Comment:
The relevant page lists them under "Class features", so changed the phrasing here to match.